System/Reader.cpp

```diff
@@ -257,7 +258,7 @@ namespace RTE {
 				if (peek == '\n') {
 					m_CurrentLine++;
 					// Only report every few lines
-					if (m_ReportProgress && (m_CurrentLine % 100 == 0)) {
+					if (m_ReportProgress && (m_CurrentLine % g_SettingsMan.LoadingScreenReportPrecision() == 0)) {
```

Comment:
It does, but if you're using this property you're smart enough not to.
Strangely enough it doesn't bork if you set a negative value.
I'm gonna keep it as is because I don't feel we need to waste computing power on safety checking it.

Reply:
Yeah, I guess x % y is the same as x % -y cause of how the maffs works. And fair enough. If you wanted to safety check it it should definitely be when it's originally read in, not when it's being used, but as you say, it's not a big deal and anyone who screws it up will figure out quick.